### DISCUSSION TITLE
Confirmed on Ryzen 5 6600H - Win 11

### DISCUSSION THREAD

**Discussion by jgetrost** [isc30/ryzen-gpu-passthrough-proxmox#58] [General]
https://www.bosgamepc.com/products/mini-pc-h66

I was able to get it working with this PC on a windows 11 VM. Still working through the bugs, but I am transcoding with jellyfin.

Only issue I ran into was I couldn't get the EFI out of my bin for a bios exe I found online. But I was able to get the ROM out of the exe. Thankfully the AMDGopDriver in the repo was able to solve my error 34 issue.

Thanks to everyone who has contributed to this endeavor.

![image](https://github.com/user-attachments/assets/e645d89a-4e18-451d-8229-cb89d2c80b72)

![image](https://github.com/user-attachments/assets/37593746-9b94-42c4-aa76-6cf7119e77c1)

**Comment by jgetrost** [comment]
I was able to get the AMDGopDriver out of the bios. I found a different version of the tools elsewhere and followed the same instructions. Might need to update the version of the tools included in the main repo.

https://github.com/jgetrost/h66-radeon-passthrough

**Reply by kimuss** [reply]
does this EfiRom.exe work on your win10? on me says: "not 64bit software compatible"... how can I start it?

**Reply by jgetrost** [reply]
> does this EfiRom.exe work on your win10? on me says: "not 64bit software compatible"... how can I start it?

Sorry, I cannot remember. Some of the things in the tool files did not work for me. I spent a few hours online doing some research and think I remember figuring out I needed a newer copy of one of the tools. I ended up downloading a zip called: "UBU_v1_79_19d" if you search for it online you can find some forums where the tools are being shared.
